public/locales/pt-BR/common.json

```diff
@@ -0,0 +1,427 @@
+  "features - support open source software": "Apoie software de código aberto",
+  "features - Sync your subscriptions, queue, and history across all your devices": "Sincroniza os podcasts que você segue, fila e histórico entre todos seus dispositivos.",
+  "features - video playback": "Reprodução de vídeo",
+  "filter from - all": "Tudo",
+  "filter from - category": "Categoria",
+  "filter from - clips": "Clipes",
+  "filter from - episodes": "Episódios",
+  "filter from - subscribed": "Seguindo",
+  "filter from - all - aria": "Filtrando por todos podcasts",
+  "filter from - category - aria": "Filtrando por categoria",
+  "filter from - subscribed - aria": "Filtrando por seguidos",
+  "filter sorting - alphabetical": "Alfabético",
+  "filter sorting - chronological": "Cronológico",
+  "filter sorting - most-recent": "Recente",
+  "filter sorting - oldest": "Mais antigo",
+  "filter sorting - random": "Aleatório",
+  "filter sorting - top-past-day": "Top - Ontem",
+  "filter sorting - top-past-week": "Top - Semana Passada",
+  "filter sorting - top-past-month": "Top - Mês Passado",
+  "filter sorting - top-past-year": "Top - Ano Passado",
+  "filter sorting - top-all-time": "Top - Desde Sempre",
+  "filter type - clips": "Clipes",
+  "filter type - playlists": "Playlists",
+  "filter type - podcasts": "Podcasts",
+  "Filtering by all podcasts": "Filtrando por todos podcasts",
+  "Filtering by category": "Filtrando por categoria",
+  "Filtering by subscribed": "Filtrando por seguidos",
+  "Finished episode": "Episódio reproduzido",
+  "Footer links": "Links do rodapé",
+  "Footer links continued": "Continuação dos links do rodapé",
+  "Forgot Password": "Esqueci minha senha",
+  "Forgot password modal": "Esqueci minha senha",
+  "Forgot?": "Esqueceu?",
+  "Full screen player": "Reprodutor de tela cheia",
+  "Funding": "Financiamento",
+  "General": "Geral",
+  "GetItOnGooglePlay": "Baixe no Google Play",
+  "Go to current live time": "Ir para transmissão ao vivo",
+  "hhmmss": "hh:mm:ss",
+  "Hide full screen player": "Ocultar o reprodutor de tela cheia",
+  "HideFilterButtons": "Ocultar os botões de filtro",
+  "HidePlaybackSpeedButton": "Ocultar o botão de velocidade de reprodução",
+  "History": "Histórico",
+  "Home page": "Página inicial",
+  "Hosts": "Hosts",
+  "How To": "Como",
+  "Inactive": "Inativo",
+  "internetConnectivityErrorMessage": "Por favor conecte-se à Internet e tente novamente.",
+  "Items": "Itens",
+  "Join our Discord": "Converse conosco no Discord",
+  "join our Discord server": "Converse conosco no Discord",
+  "Join our XMPP server": "Junte-se ao nosso servidor XMPP",
+  "Jump back 1 second": "Voltar 1 segundo",
+  "Jump back 10 seconds": "Voltar 10 segundos",
+  "Jump forward 1 second": "Avançar 1 segundo",
+  "Jump forward 30 seconds": "Avançar 30 segundos",
+  "Language": "Idioma",
+  "language - en": "English",
+  "language - es": "Spanish",
+  "language - lt": "Lithuanian",
```

Comment:
I keept these as is. Should I translate it to Portuguese?
Should I add the missing languages here?

*https://github.com/podverse/podverse-web/blob/d3274004c03f7e8575bd2419aef67687cd81dc93/next-i18next.config.js

Reply:
Yeah, I think that'd be great 👍

Reply:
a735e321